src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -93,6 +93,7 @@
     <PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="3.1.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.2" />
+    <PackageReference Include="Uno.Core" Version="2.3.0" />
```

Comment:
do we need to add explicit references to uno.core?

Reply:
Uno.Core is not really useful. We'd like to remove it ideally. It's included in Uno.UI, so we don't need an explicit reference.